ykman-gui/qml/OtpYubiOtpView.qml

```diff
@@ -191,31 +147,17 @@ ColumnLayout {
                 flat: true
                 Layout.alignment: Qt.AlignLeft | Qt.AlignBottom
             }
-            Row {
-                id: row
-                spacing: 5
-                Layout.alignment: Qt.AlignRight | Qt.AlignBottom
-                CheckBox {
-                    id: enableUpload
-                    text: qsTr("Upload")
-                    Layout.alignment: Qt.AlignRight | Qt.AlignBottom
-                    ToolTip.delay: 1000
-                    font.pixelSize: constants.h3
-                    ToolTip.visible: hovered
-                    ToolTip.text: qsTr("Upload credential to YubiCloud (opens a web browser)")
-                    Material.foreground: yubicoBlue
-                }
 
-                FinishButton {
-
-                    onClicked:  finish()
-                    enabled: publicIdInput.acceptableInput
-                               && privateIdInput.acceptableInput
-                               && secretKeyInput.acceptableInput
-                    toolTipText: qsTr("Finish and write the configuration to the YubiKey")
-                    Layout.alignment: Qt.AlignRight | Qt.AlignBottom
-                }
+            FinishButton {
+                Layout.fillWidth: false
+                Layout.alignment: Qt.AlignRight | Qt.AlignBottom
 
+                onClicked:  finish()
+                enabled: publicIdInput.acceptableInput
+                            && privateIdInput.acceptableInput
+                            && secretKeyInput.acceptableInput
+                toolTipText: qsTr("Finish and write the configuration to the YubiKey")
+                Layout.alignment: Qt.AlignRight | Qt.AlignBottom
```

Comment:
The qt now complains that the property is set multiple times:
```
qrc:/qml/OtpYubiOtpView.qml:160:17: Property value set multiple times
```
so I think this one can be removed.

This lead to issues described in the last comments of #357 in Fedora, where I pulled this patch to.